plugins/Installation/lang/en.json

```diff
@@ -137,6 +137,9 @@
         "SystemCheckPhpSetting": "To prevent some critical issue, you must set the following in your php.ini file: %s",
         "SystemCheckUpdateHttps": "Update over HTTPS",
         "SystemCheckUpdateHttpsNotSupported": "Matomo cannot use HTTPS to update, it will fall back to the insecure HTTP update. Check that CURL or allow_url_fopen is supported and that the openssl PHP extension is installed: https:\/\/matomo.org\/faq\/troubleshooting\/faq_177\/.",
+        "SystemCheckProcessorFormat" : "OS Format",
```

Comment:
I'm not sure if OS Format is a well known term (a search only shows results on how to format disks).
As I don't know any better term myself, we could maybe instead use "PHP Binary" or something like this.